articles/python/azure-sdk-sovereign-domain.md

```diff
@@ -23,28 +23,32 @@ Pre-defined sovereign cloud constants are provided by the `azure_cloud` module o
 
 To use a definition, import the appropriate constant from `msrestazure.azure_cloud` and apply it when creating client objects. 
 
-When using `DefaultAzureCredential`, as shown in the following example, you also need to use the appropriate value from `azure.identity.AzureAuthorityHosts`.
+When using `DefaultAzureCredential`, as shown in the following example, you also need to use the appropriate value from `CLOUD.endpoints.active_directory`.
 
 ```python
 import os
-from msrestazure.azure_cloud import AZURE_CHINA_CLOUD as cloud
+from msrestazure.azure_cloud import AZURE_CHINA_CLOUD as CLOUD
 from azure.mgmt.resource import ResourceManagementClient, SubscriptionClient
 from azure.identity import DefaultAzureCredential
 
-# Assumes the subscription ID to use is in the AZURE_SUBSCRIPTION_ID environment variable
+# Assumes the subscription ID and tenant ID to use are in the AZURE_SUBSCRIPTION_ID,
+# AZURE_TENANT_ID environment variables
 subscription_id = os.environ["AZURE_SUBSCRIPTION_ID"]
+tenant_id = os.environ["AZURE_TENANT_ID"]
 
 # When using sovereign domains (that is, any cloud other than AZURE_PUBLIC_CLOUD),
 # you must use an authority with DefaultAzureCredential.
-credential = DefaultAzureCredential(authority=cloud.endpoints.active_directory)
+credential = DefaultAzureCredential(authority=CLOUD.endpoints.active_directory, tenant_id=tenant_id)
 
-resource_client = ResourceManagementClient(credential,
-    subscription_id, base_url=cloud.endpoints.resource_manager,
-    credential_scopes=[cloud.endpoints.resource_manager + ".default'"])
+resource_client = ResourceManagementClient(
+    credential, subscription_id,
+    base_url=CLOUD.endpoints.resource_manager,
+    credential_scopes=[CLOUD.endpoints.resource_manager + "/.default"])
```

Comment:
The `/` before `.default` is always necessary. Otherwise `AZURE_CHINA_CLOUD`'s ARM scope will be constructed as `https://management.chinacloudapi.cn.default`.

https://github.com/Azure/msrestazure-for-python/blob/8849f398b6ebd4607de63c2f5d1318f44ec1d822/msrestazure/azure_cloud.py#L156

```py
        resource_manager='https://management.chinacloudapi.cn',
```

For some resource, double slashes are mandatory: `https://management.azure.com//.default`.

See AAD doc: [Trailing slash and /.default](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent#trailing-slash-and-default)